src/client/graphics/layers/PlayerPanel.ts

```diff
@@ -146,7 +146,15 @@ export class PlayerPanel extends LitElement implements Layer {
   }
 
   tick() {
-    this.requestUpdate();
+    if (this.isVisible && this.tile) {
+      const myPlayer = this.g.myPlayer();
+      if (myPlayer && myPlayer.isAlive()) {
+        myPlayer.actions(this.tile).then((actions) => {
+          this.actions = actions;
+          this.requestUpdate();
+        });
```

Comment:
You could make this function `async tick()` and then you'd be able to use `await` here:

```suggestion
        this.actions = await myPlayer.actions(this.tile);
        this.requestUpdate();
```